src/components/ThreadEnvelope.vue

```diff
@@ -310,6 +318,15 @@ export default {
 				(tag) => tag.imapLabel !== '$label1' && !(tag.displayName.toLowerCase() in hiddenTags)
 			)
 		},
+		hasChangedSubject() {
+			return this.cleanSubject !== this.cleanThreadSubject
+		},
+		cleanSubject() {
+			return this.envelope.subject.replace(/((?:[\t ]*(?:R|RE|F|FW|FWD):[\t ]*)*)/i, '')
+		},
+		cleanThreadSubject() {
+			return this.threadSubject.replace(/((?:[\t ]*(?:R|RE|F|FW|FWD):[\t ]*)*)/i, '')
```

Comment:
Please move the cleaning logic to a reusable method. You could name it `filterSubject()`. Otherwise, the code is duplicated.